src/antidote_crdt_orset.erl

```diff
@@ -108,33 +108,33 @@ value({fragment, Elem}, ORSet) ->
 value({tokens, Elem}, ORSet) ->
     case orddict:find(Elem, ORSet) of
         error ->
-            orddict:new();
+            [];
```

Comment:
I think it's nicer to use `orddict:new` here and be explicit about the used data structure